README.md

```diff
@@ -33,9 +33,36 @@ Globalization should be taken into account at every step.
 ## Setting up your dev environment
 
 1. Clone this repository.
-2. Build and run the `SampleWebApp` project.
-3. From the admin, enable the module's only feature.
-4. (optional) Start using the features, by creating a new `Product` content type, and adding the product part to it.
+2. Build and run the `OrchardCore.Commerce.Web` project, then run the `OrchardCore Commerce - Development` recipe on the setup screen.
+3. Go to *Features*, search for “Commerce” and turn on everything.
+4. Go to *Settings → Commerce*, and save it (by default, this setting has no value, so you need to save it, it’s a bug, and it will need to be fixed in the future, it can break the product sites).
+5. Go to *Settings → Stripe API*. Set the keys (test keys can be found below). If the keys are not set, payment won't work.
+6. Go to Content Items, and create a `Product`.
+
+## Test data for Stripe Payment
+
+### API keys
+
+Stripe API uses a secret-publishable key pair. The following API keys are public sample test keys, they can be found in [Stripe's documentation](https://stripe.com/docs/keys#obtain-api-keys).
+
+* Publishable key: `pk_test_51H59owJmQoVhz82aWAoi9M5s8PC6sSAqFI7KfAD2NRKun5riDIOM0dvu2caM25a5f5JbYLMc5Umxw8Dl7dBIDNwM00yVbSX8uS`
+* Secret key: `sk_test_51H59owJmQoVhz82aOUNOuCVbK0u1zjyRFKkFp9EfrqzWaUWqQni3oSxljsdTIu2YZ9XvlbeGjZRU7B7ye2EjJQE000Dm2DtMWD`
+
+**These are just test keys. Don’t submit any personally identifiable information in requests made with this key.**
+
+The Stripe API key pair can be set inside _Dashboard → Settings → Stripe API_.
+
+### Cards
+There are available test cards that can be found in [Stripe's documentation](https://stripe.com/docs/testing).
+
+There are multiple test cards that can simulate any scenario, including error codes. Here are two examples:
+
+| Brand |      Number	     |      CVC     |       Date      |     Result                        |
+| ----- | ---------------- | ------------ | --------------- | --------------------------------- |
+| Visa  | 4242424242424242 | Any 3 digits | Any future date | Success                           |
+| Visa  | 4000000000009995 | Any 3 digits | Any future date | card_declined, insufficient_funds |
+
+You can also test 3DS authentication with: `4000000000003220`.
```

Comment:
It just occurred to me, but why did you not include the card number for testing 3DS in the table as well?

Reply:
There were a lot of table types on Stripes's page and each had a different header, so it wouldn't fit here. But yes, we could put it there and for the `Result` something like `Success but 3DS Required`.